src/components/ActionDetailsList.tsx

```diff
@@ -228,38 +228,38 @@ export default connect<typeof stateProps, {}, ReceivedProps>(mapStateToProps, ma
 function getActionPayloadRenderer(action: CLM.ActionBase, component: ActionDetailsList, isValidationError: boolean) {
     if (action.actionType === CLM.ActionTypes.TEXT) {
         const textAction = new CLM.TextAction(action)
-        return (<ActionPayloadRenderers.TextPayloadRendererContainer
```

Comment:
This pattern follows for most of the components. Switch from `Container` to `WithHighlights` and add new property to control marking of missing entities. (ActionDetailList no memory so there is no concept of missing, but in TrainDialog where memory exists we do need to know this)